source/a_releasenotes.rst

```diff
@@ -2,6 +2,24 @@
 Release notes
 *************
 
+Release 3Di live site
```

Comment:
We need to make this more consistent over all the release titles. Some have a - in between, the last three do not. Either all with a -, or remove the dashes from previous releases.